File: src/services/get_def/findRefsUtils.ml
(*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *)

type ast_info = (Loc.t, Loc.t) Flow_ast.Program.t * File_sig.t * Docblock.t

type type_info =
  Context.t
  * (ALoc.t, ALoc.t * Type.t) Flow_ast.Program.t
  * Type.Properties.Set.t Loc_collections.LocMap.t

File: rust_port/crates/flow_services_get_def/src/find_refs_utils.rs
/*
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

use std::sync::Arc;

use flow_common::docblock::Docblock;
use flow_parser::ast;
use flow_parser::loc::Loc;
use flow_parser_utils::file_sig::FileSig;

pub type AstInfo = (Arc<ast::Program<Loc, Loc>>, Arc<FileSig>, Arc<Docblock>);
